draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -790,7 +792,7 @@ Further details of the algorithm are given in the following sections.
 
 ## Message identifiers
 
-### Primary ID (required)
```

Comment:
Because the () mucks up the Markdown section referencing. It says it can't find the reference. I think you can specify an explicit referencing target.

I've updated the section wording to make it clear that a Primary ID is always constructed.